src/socket.rs

```diff
@@ -394,21 +394,22 @@ impl NlSocketHandle {
     /// Convenience function for resolving a [`str`] containing the
     /// generic netlink family name to a numeric generic netlink ID.
     pub fn resolve_genl_family(&mut self, family_name: &str) -> Result<u16, NlError> {
+        let mut res = Err(NlError::new(format!(
+            "Generic netlink family {} was not found",
```

Comment:
I like to do stuff like this in `'{}'`. but I guess that's a personal favourite, I'm fine with either way